deepchem/rl/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Interface for reinforcement learning."""
+# flake8: noqa
 try:
     from deepchem.rl.a2c import A2C  # noqa: F401
     from deepchem.rl.ppo import PPO  # noqa: F401
+    from deepchem.rl import A2C  # noqa: F401
```

Comment:
You will have a namespace conflict. We want to keep TF for default now, but makek torch a2c visible

Reply:
Making torch_a2c visible is making 'torch' undefined. Trying to figure a work around.

Reply:
Let's remove the top level pytorch import